### PR TITLE
HiveMigrator - CLI operation

### PR DESCRIPTION
**Pull request by benradstone**

### CODE REVIEW

File: docs/operation-cli.md [hunk 1]
```diff
@@ -222,6 +230,106 @@ Migrations can be in one of eight states:
...
+| [`hive agent show`](./command-reference.md#hive-agent-show) | Show the configuration for a hive agent |
+| [`hive agent types`](./command-reference.md#hive-agent-types) | List supported hive agent types |
+
+## Define hive rules
+
+### Add hive rules
+
+Define which databases and tables you want to migrate by creating hive rules.
+
+Create a database pattern and a table pattern using [regex](https://regex101.com/) that will match the databases and tables you want to migrate. For example, using `--database-pattern test*` will match any database that has "test" at the beginning of its name, such as `test01`, `test02`, `test03`.
```
Comment (lucasdigital) [inline docs/operation-cli.md:275]: very minor suggestion - change "that has" to "with".

**Review by lucasdigital — APPROVED** [review]
All looks good. Made one tiny reword suggestion but the current wording is fine, really. 

I'm a tiny bit concerned that "hive" as a small-caps command word might be applied in places where, strictly speaking, it should be "Hive". However, the docs are consistent and may read better for that.

File: docs/operation-cli.md (hunk 1 above)
Comment (benradstone) [reply docs/operation-cli.md]: Change made,

**Comment by benradstone** [comment]
> All looks good. Made one tiny reword suggestion but the current wording is fine, really.
> 
> I'm a tiny bit concerned that "hive" as a small-caps command word might be applied in places where, strictly speaking, it should be "Hive". However, the docs are consistent and may read better for that.

I would agree if the context was always in reference to Apache Hive, but it isn't. I feel as though the product shouldn't have 'hive' in its name at all, as it functions across multiple database platforms.

**Pull request merged**